lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -115,6 +115,34 @@ describe('ViewerComponent', () => {
 
     });
 
+    describe('originalMimeType', () => {
+
+        it('should set mimeType input correctly', () => {
+            const mimeTypeValue = 'image/png';
+            component.mimeType = mimeTypeValue;
```

Comment:
component.mimeType is being set to equal mimeTypeValue and in the same test same thing is being expected, it is only going to resolve to true.

Reply:
I have modified the test cases